Wpstandards

joelrd opened the pull request:
WP Standarization for admin\aioseop_module_class.php up to line 2588

michaeltorbert commented:
@joelrd @amostajo There are still a lot of removed comments. Are these on purpose for a reason or was it just a mistake?

joelrd commented:
@michaeltorbert @amostajo Sorry, I just noticed that I didn't add all the previous comments missing from the last Pull-Request. I double checked and added them again. If there are other missing comments let me know. I am really sorry. Because I added a comment on every loop by my own, but I didn't know how it was structured and it was taking more time than expected. So I decided to delete them. Some of the comments were confused with those. I left this practice on a couple of files after. I will check them again and make sure that none of the comments are missing for the next files.
